vllm/model_executor/models/falcon.py

```diff
@@ -391,6 +391,19 @@ def forward(
 
 class FalconForCausalLM(nn.Module, SupportsPP):
 
+    bitsandbytes_stacked_params_mapping = {}
+
+    default_bitsandbytes_target_modules = [
+        ".query_key_value.",
+        ".dense.",
+        ".dense_h_to_4h.",
+        ".dense_4h_to_h.",
+    ]
+
+    # BitandBytes specific attributes
+    # in TP, these weights are partitioned along the column dimension (dim=-1)
+    column_parallel_weights_modules = [".dense_4h_to_h.", ".dense."]
```

Comment:
Shouldn't all of these be put under the `# BitandBytes specific attributes` comment?

Reply:
updated per comments.